client/src/pages/Apply/Apply.js

```diff
@@ -205,15 +185,10 @@ const Apply = ({ navigation }) => {
                     returnKeyType="done"
                     onChangeText={(text) => setName(text)}
                   />
-                  {'name' in errors ? (
-                    <FormControl.ErrorMessage>
-                      Required.
-                    </FormControl.ErrorMessage>
-                  ) : null}
                 </View>
               </FormControl>
 
```

Comment:
It looks like you are doing `<FormControl> <Input/> <FormControl>  <FormControl> <Input/> <FormControl/>`. What if you merge the two form controls into `<FormControl> <Input/> <Input/>  <FormControl>`